nifi-nar-bundles/nifi-hbase-bundle/nifi-hbase-processors/pom.xml

```diff
@@ -82,5 +90,15 @@
             </exclusions>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.nifi</groupId>
+            <artifactId>nifi-mock-record-utils</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.hbase</groupId>
+            <artifactId>hbase-common</artifactId>
```

Comment:
We should keep all the HBase client dependencies behind HBaseClientService so that the version of the client doesn't leak into the processors NAR. 

It looks like the main reason for adding this was to use the `Bytes` class from HBase common, which we ran into once before and we ended exposing some `toBytes` methods on `HBaseClientService`. We should add the additional toBytes methods that we need, or we could possibly add a method like:

` byte[] asBytes(String field, RecordFieldType fieldType, Record record)`